control/tests/baseline_images/freqresp_test/gangof4-pvtol-expected.png

```diff
@@ -0,0 +1 @@
+/Users/murray/Dropbox/macosx/src/python-control/murrayrm/control/tests/baseline_images/freqresp_test/gangof4-pvtol.png
```

Comment:
?

Reply:
That's very odd.  Should have been a copy of the image...

Reply:
Symbolic link set up by pytest that apparently used an absolute path.  Fixed.